add subdaily granularity (#5882)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/metrics-overview.md

  
Once you've created your semantic models, it's time to start adding metrics. Metrics can be defined in the same YAML files as your semantic models, or split into separate YAML files into any other subdirectories (provided that these subdirectories are also within the same dbt project repo).

> This page explains the different supported metric types you can add to your dbt project. The keys for metrics definitions are:

Reviewer: ```suggestion
This article explains the different supported metric types you can add to your dbt project. The keys for metrics definitions are:
```

      {{  Dimension('entity__name') }} > 0 and {{ Dimension(' entity__another_name') }} is not
      null and {{ Metric('metric_name', group_by=['entity_name']) }} > 5
```
</File>

</VersionBlock>

import SLCourses from '/snippets/_sl-course.md';

<SLCourses/>

> ## Default granularity for metircs

Reviewer 2: @mirnawong1 This whole section called "Default granularity for metrics" should be version blocked to 1.9+.
Also noting that "metrics" is misspelled in the title (though maybe that's already fixed in production!)

> It's possible to define a default time granularity for metrics that differs from the granularity of the default aggregation time dimensions (`metric_time`). This is useful if your time dimension has a very fine grain, like second or hour, but you typically query metrics rolled up at a coarser grain. The granularity can be set using the `time_granularity` parameter on the metric and defaults to `day`.

Reviewer 2: Would note that while it defaults to day, if day is not available because the dimension is defined at a coarser granularity, it will default to the defined granularity for the dimension!

> It's possible to define a default time granularity for metrics if it's different from the granularity of the default aggregation time dimensions (`metric_time`). This is useful if your time dimension has a very fine grain, like second or hour, but you typically query metrics rolled up at a coarser grain. The granularity can be set using the `time_granularity` parameter on the metric, and defaults to `day`.

Reviewer: ```suggestion
It's possible to define a default time granularity for metrics that differs from the granularity of the default aggregation time dimensions (`metric_time`). This is useful if your time dimension has a very fine grain, like second or hour, but you typically query metrics rolled up at a coarser grain. The granularity can be set using the `time_granularity` parameter on the metric and defaults to `day`.
```

### Example

> I have a semantic model called `orders` with a time dimension called `order_time`. I want the `orders` metric to roll up to `monthly` by default; however, I want the option to be able to look at these metrics hourly. I can set the `time_granularity` parameter on the `order_time` dimension to `hour`, and then set the `time_granularity` parameter in the metric to `month`.

Reviewer: ```suggestion
You have a semantic model called `orders` with a time dimension called `order_time`. You want the `orders` metric to roll up to `monthly` by default; however, you want the option to look at these metrics hourly. You can set the `time_granularity` parameter on the `order_time` dimension to `hour`, and then set the `time_granularity` parameter in the metric to `month`.
```

  {{ TimeDimension('time_dimension', 'granularity') }}

> filter: |  

Reviewer 2: @mirnawong1 Can we version block this metric filter example to versions 1.8+?

 {{ Metric('metric_name', group_by=['entity_name']) }}  # Available in v1.8 or with [versionless (/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) dbt Cloud.
```

</File>

> For example, if I wanted to filter for the order date dimension, grouped by month, I'd use the following syntax:

Reviewer: ```suggestion
For example, if you want to filter for the order date dimension grouped by month, use the following syntax:
```
